CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @rdfjs/types
 
+## 2.0.0
```

Comment:
Please revert changes to the changeling and create a changeset instead using `npx changeset` command

Also, are you sure this warrants a major bump?

Reply:
As per @jeswr 's comment and @rubensworks 's 👍 I think it does.

If the methods were optional in the spec (just like the variable method, I could be in favour of that), then a minor bump would suffice, but we would need to get consensus on that.

Reply:
Changelog reverted and `changeset` added: https://github.com/rdfjs/types/pull/46/commits/14906cfc5f820fa530cf2901860193ed350e0673.